content/post/081-ethdenver-2020.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2020-02-06
+url: 081-ethdenver-2020
+title: Hack with us at ETHDenver
+author: Jenn Turner
+---
+
+![IPFS at ETHDenver 2020](static/img/081-ethdenver-2020/ipfs-ethdenver.png)
+
+Next week, the IPFS team will be heading to Denver, Colorado for [ETHDenver](https://www.ethdenver.com/), one of the biggest Ethereum and decentralized web developer-focused events in the US. We’re very excited to connect with the dweb community during a week full of presentations, workshops, [#BUIDLathon](https://www.ethdenver.com/buidlweek/)-ing, and community gathering.
+
+On Thursday, February 13, the IPFS Community is hosting the [Distributed Networks Summit](http://www.eventbrite.com/e/distributed-networks-summit-ipfs-friends-tickets-86959928487?aff=protocollabs), featuring speakers and workshops from IPFS Core, Protocol Labs, Textile, ProtoSchool, 3Box, RTrade Technologies, Pinata, Althea, Quorum Control, Kauri, and more! [Register to attend](http://www.eventbrite.com/e/distributed-networks-summit-ipfs-friends-tickets-86959928487?aff=protocollabs) so you can figure out how to implement IPFS in your dapp, with the advice from leading experts. Think of it as our early Valentine’s Day gift from us to you. 💝
```

Comment:
Suggest ending with "from the IPFS community to you" because the community set up the event in entirety.